.travis.yml

```
@@ -68,13 +72,14 @@ addons:
     - ccache
     - python
     - pip
+    - automake
 install: |
     if [[ $TRAVIS_OS_NAME == linux && "$TRAVIS_BRANCH" != "master" ]]; then 
       bash tools/ci/check-commit.sh; 
     fi
 script: |
     mkdir -p build && cd build && CC=/usr/bin/clang CXX=/usr/bin/clang++ cmake .. && \
-    make -j2 && cd ../tools && bash ci/ci_check.sh
+    make && cd ../tools && bash ci/ci_check.sh
```

```suggestion
    make -j2 && cd ../tools && bash ci/ci_check.sh
```
I think, this should not be changed.